test/system/252-quadlet.bats

```diff
@@ -221,12 +221,11 @@ EOF
 }
 
 @test "quadlet conflict names" {
-    skip "FIXME: #24047, temporary skip because this is an intense flake"
-
     # If two directories in the search have files with the same name, quadlet should
     # only process the first name
-    dir1=$PODMAN_TMPDIR/$(random_string)
-    dir2=$PODMAN_TMPDIR/$(random_string)
+    # prepend the directory name to make test that the order set in QUADLET_UNIT_DIRS is kept
+    dir1=$PODMAN_TMPDIR/02_$(random_string)
+    dir2=$PODMAN_TMPDIR/01_$(random_string)
```

Comment:
I was thinking about this yesterday... and think the original form, without 02/01, is preferable. Your way would catch deterministic sorting; the original way catches a broader class of failures (albeit at the cost of them being flaky and hard to debug).

Would you mind fixing `Image` as I suggested in the bug report, at least in the dir2 file? Something like `Image=YOU-SHOULD-NEVER-SEE-THIS-BAD-BAD-BAD`? Just to help future maintainers?

Reply:
Thanks for the review. Done and Done.